### ISSUE TITLE
ZSUN hardware compatibility

### ISSUE THREAD

**Issue by LucaBongiorni** [agentsac/AR150pineapple#1]
Hey,
Did you maybe have the chance to install pineapple nano fw inside a ZSUN?
https://wiki.hackerspace.pl/projects:zsun-wifi-card-reader

**Comment by agentsac** [comment]
No, I don't own a ZSUN. Looks like a neat little device, however the hardware seems limited compared to the wifi pineapple nano. The addition of an SD card reader is nice, but only having one radio would limit the available features. For the GL-AR150, there is a USB port where one could add a USB Alfa wireless card as a second radio for the Pine AP features. There is definitely potential in the ZSUN, but not with the wifi pineapple nano firmware as-is.

**Comment by LucaBongiorni** [comment]
Hi,
I completely agree with you about its limitations regarding a 2nd radio RTX.

Just in case you manage to succesfully push the Nano fw into it... let me
know :)

I was trying, but without success.

Cheers,
Luca
